test/gtest/common/googletest/gtest-death-test.cc

```diff
@@ -30,12 +30,12 @@
 //
 // This file implements death tests.
 
-#include "gtest/gtest-death-test.h"
+#include "gtest-death-test.h"
```

Comment:
maybe simple to add "gtest" to include path?

Reply:
Do you mean `#include <common/googletest/gtest-death-test.h>` ?

Reply:
ok